webapp/src/selectors.js

```diff
@@ -17,3 +19,34 @@ export const getMessage = (state) => {
     return post.message;
 };
 export const getItems = (state) => getPluginState(state).items;
+export const getCurrentTeamRoute = (state) => {
+    const config = getConfig(state);
+
+    const basePath = getSiteURL();
+    const teamName = state.entities.teams.teams[state.entities.teams.currentTeamId].name;
+
+    return basePath + '/' + teamName + '/';
+}
+
+function getSiteURL() {
```

Comment:
I am a webapp kook, but isn't there a way to get it from a mattermost-provided API?

Reply:
I am not sure how to do this, since the function I copied is from mattermost-webapp, and I am not sure whether it can be imported or how (I tried but I was not able).

If you can direct on how to do this, I am more than happy to make the change.